[2025-02-23T15:52:44.429+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-23T15:52:44.451+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_first_dag.second_task scheduled__2025-02-21T00:00:00+00:00 [queued]>
[2025-02-23T15:52:44.461+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_first_dag.second_task scheduled__2025-02-21T00:00:00+00:00 [queued]>
[2025-02-23T15:52:44.462+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 6
[2025-02-23T15:52:44.476+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): second_task> on 2025-02-21 00:00:00+00:00
[2025-02-23T15:52:44.485+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1567) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-23T15:52:44.485+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'my_first_dag', 'second_task', 'scheduled__2025-02-21T00:00:00+00:00', '--job-id', '62', '--raw', '--subdir', 'DAGS_FOLDER/my_first_dags.py', '--cfg-path', '/tmp/tmpvlyvcxik']
[2025-02-23T15:52:44.488+0000] {standard_task_runner.py:105} INFO - Job 62: Subtask second_task
[2025-02-23T15:52:44.488+0000] {standard_task_runner.py:72} INFO - Started process 1578 to run task
[2025-02-23T15:52:44.537+0000] {task_command.py:467} INFO - Running <TaskInstance: my_first_dag.second_task scheduled__2025-02-21T00:00:00+00:00 [running]> on host 2cebbf768e94
[2025-02-23T15:52:44.636+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='gzang' AIRFLOW_CTX_DAG_ID='my_first_dag' AIRFLOW_CTX_TASK_ID='second_task' AIRFLOW_CTX_EXECUTION_DATE='2025-02-21T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-21T00:00:00+00:00'
[2025-02-23T15:52:44.637+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-23T15:52:44.651+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-02-23T15:52:44.653+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hey, this is task 2, and will be run after task 1.']
[2025-02-23T15:52:44.661+0000] {subprocess.py:99} INFO - Output:
[2025-02-23T15:52:44.662+0000] {subprocess.py:106} INFO - hey, this is task 2, and will be run after task 1.
[2025-02-23T15:52:44.663+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-02-23T15:52:44.687+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-23T15:52:44.688+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=my_first_dag, task_id=second_task, run_id=scheduled__2025-02-21T00:00:00+00:00, execution_date=20250221T000000, start_date=20250223T155244, end_date=20250223T155244
[2025-02-23T15:52:44.747+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-23T15:52:44.771+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-23T15:52:44.773+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-02-24T04:47:22.722+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-24T04:47:22.770+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_first_dag.second_task scheduled__2025-02-21T00:00:00+00:00 [queued]>
[2025-02-24T04:47:22.795+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_first_dag.second_task scheduled__2025-02-21T00:00:00+00:00 [queued]>
[2025-02-24T04:47:22.796+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 6
[2025-02-24T04:47:22.858+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): second_task> on 2025-02-21 00:00:00+00:00
[2025-02-24T04:47:22.874+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1131) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-24T04:47:22.878+0000] {standard_task_runner.py:72} INFO - Started process 1145 to run task
[2025-02-24T04:47:22.880+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'my_first_dag', 'second_task', 'scheduled__2025-02-21T00:00:00+00:00', '--job-id', '99', '--raw', '--subdir', 'DAGS_FOLDER/my_first_dags.py', '--cfg-path', '/tmp/tmpywi06e4x']
[2025-02-24T04:47:22.883+0000] {standard_task_runner.py:105} INFO - Job 99: Subtask second_task
[2025-02-24T04:47:22.957+0000] {task_command.py:467} INFO - Running <TaskInstance: my_first_dag.second_task scheduled__2025-02-21T00:00:00+00:00 [running]> on host 79e0cb3ed452
[2025-02-24T04:47:23.066+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='gzang' AIRFLOW_CTX_DAG_ID='my_first_dag' AIRFLOW_CTX_TASK_ID='second_task' AIRFLOW_CTX_EXECUTION_DATE='2025-02-21T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-21T00:00:00+00:00'
[2025-02-24T04:47:23.067+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-24T04:47:23.084+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-02-24T04:47:23.085+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hey, this is task 2, and will be run after task 1.']
[2025-02-24T04:47:23.096+0000] {subprocess.py:99} INFO - Output:
[2025-02-24T04:47:23.097+0000] {subprocess.py:106} INFO - hey, this is task 2, and will be run after task 1.
[2025-02-24T04:47:23.098+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-02-24T04:47:23.122+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-24T04:47:23.123+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=my_first_dag, task_id=second_task, run_id=scheduled__2025-02-21T00:00:00+00:00, execution_date=20250221T000000, start_date=20250224T044722, end_date=20250224T044723
[2025-02-24T04:47:23.177+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-24T04:47:23.235+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-24T04:47:23.257+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
